recipes/bioconductor-a4/meta.yaml

```diff
@@ -1,47 +1,52 @@
-{% set version = "1.50.0" %}
+{% set version = "1.54.0" %}
```

Comment:
I'm wondering in general why the ordering has changed. I remember that we fixed the ordering of YAML sections at some point to make the diffs easier.